src/index.html

```diff
@@ -8,10 +8,16 @@
       name="viewport"
       content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no"
     />
```

Comment:
```suggestion
    />
    <meta name="description" content="This is an example of a meta description. This will often show up in search results.">
```
Up your SEO game by adding in a meta description -- they may be included in search results to concisely summarize page content.

For more info, check out this article: https://moz.com/blog/seo-meta-tags